src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,27 @@
+import cn from 'classnames';
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo,
+  isLoadingUser: boolean,
+  selectedUser: User | null,
+  onClosePopUp: () => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
```

Comment:
i'd de-structure it deeper since you're using only `id` and completed `props`
```suggestion
  selectedTodo: {
    id, completed,
  },
```